clar2wasm/src/deserialize.rs

```diff
@@ -579,16 +592,16 @@ impl WasmGenerator {
         memory: MemoryId,
         offset_local: LocalId,
         end_local: LocalId,
-        ok_ty: &TypeSignature,
-        err_ty: &TypeSignature,
+        offset_result: LocalId,
+        response_type: ResponseTypePair,
```

Comment:
What motivated that change?

Reply:
Clippy was throwing an error for too many arguments in a function. This resulted in the failure of the CI.

Reply:
Pffff... for an internal function, a new struct is overkill in my opinion. We should just ignore that lint here.